### ISSUE TITLE
Adapt to different image sizes

### ISSUE THREAD

**Issue by Breeze-Zero** [libffcv/ffcv#340]
My task requires that the images do not scale and crop uniformly, so I can only batch size 1 to import different sizes of data. Can FFCV writer achieve this?

**Comment by tavisshore** [comment]
Have you used the `RandomResizedCrop()` function?

**Comment by Breeze-Zero** [comment]
> Have you used the `RandomResizedCrop()` function, or `ResizedCropRGBImageDecoder()` class?

No, Just reading the data, the data itself is not consistent and I don't want to resize it.

**Comment by Breeze-Zero** [comment]
> What are you actually trying to do

Here's the size range of my data: {(448, 132, 10, 9, 12), (448, 132, 10, 10, 12), (512, 204, 10, 11, 12), (512, 204, 10, 12, 12), (512, 204, 10, 13, 12). (512, 246, 10, 11, 12), (512, 204, 10, 9, 12), (512, 204, 10, 10, 12), (512, 246, 10, 9, 12), (512, 162, 10, 9, 12), (512, 246, 10, 10, 12), (512, 162, 10, 10, 12), (512, 246, 10, 12, 12)}, some dimensions are not consistent. I can't use cropping or scaling operations due to the peculiarity of the data, so I have to read it into the dataloader. The model is trained in a batch size. Whether the Writer of FFCV can support this kind of data without changing the size of the data?
